ofl/notoserifhk/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,8 @@
+<p>
+  Noto is a global font collection for writing in all modern and ancient
+  languages. Noto Serif HK is an modulated (“serif”) design for languages
+  in Hong Kong that use the <em>Traditional Chinese</em> variant of the Han
+  ideograms. It also supports
+  <em>Latin, Cyrillic, Greek, Katakana, Hiragana</em> and <em>Hangul</em>. It
+  has multiple weights.
```

Comment:
Maybe something like "It is a variable font with a weight axis ranging from ExtraLight to ExtraBlack".